apitests/spec/device_rules_spec.ts

```diff
@@ -1,10 +1,10 @@
 import request = require('request');
+var uuid = require('node-uuid');
 
-
-describe('device rules api', () => {
+xdescribe('device rules api', () => {
```

Comment:
since you tagged this as "x", you don't have to tag its nested elements "x". Everything inside it will be disabled.